HACKING.md

```diff
@@ -0,0 +1,95 @@
+# Hacking
+
+## Quickstart
+
+Use [ghcup][ghcup] to install the `cabal` cli tool and the ghc version we're using:
+
+```sh
+$ ghcup install ghc 8.10
+$ ghcup set ghc 8.10
+$ ghcup install cabal 
```

Comment:
ghcup automatically installs cabal and sets it as active -- so we can remove this and the next line
```suggestion
```